.github/auto-comment.yml

```diff
@@ -0,0 +1,43 @@
+# Comment to a new issue.
+# Disabled
+# issueOpened: ""
+
+pullRequestOpened: >
+
+  Thank you for submitting this PR!
+
+  A maintainer will be here shortly to review it.
+
+  We are super grateful to get code contributions and aim to make the best of
+  the contribution process. However, we are also overloaded with work, we'd be
+  happy if you can make sure that:
+
+    * The context for this PR is cleared, with relevant discussion, decisions
+      and stakeholders linked/mentioned.
+
+    * You have self-reviewed yourself. Please make a self-review and leave
```

Comment:
```suggestion
    * You have self-reviewed it yourself. Please make a self-review and leave
```

Reply:
I'd also make it clear that _most_ motivation should go in code comments. Self-reviews are just for additional motivation related to the _change_ that wouldn't make much sense as a code comment.